### PR TITLE
buffs drones

### PR DESCRIPTION
**Pull request by ghost**
drones have better gear to start
#### Changelog

:cl:
rscadd: Drones now have soap.
rscadd: Drones now have RCDs and RPDs.
rscadd: Drone welders are now better.
/:cl:

### CODE REVIEW

File: code/game/objects/items/weapons/storage/backpack.dm [hunk 1]
```diff
@@ -334,11 +334,15 @@
...
-	new /obj/item/weapon/weldingtool(src)
+	new /obj/item/weapon/weldingtool/hugetank(src)
 	new /obj/item/weapon/crowbar(src)
 	new /obj/item/stack/cable_coil/random(src)
 	new /obj/item/weapon/wirecutters(src)
 	new /obj/item/device/multitool(src)
+	new /obj/item/device/lightreplacer(src)
+	new /obj/item/weapon/soap/syndie(src)
+	new /obj/item/weapon/rcd/rcd
+	new /obj/item/weapon/pipe_dispenser/rpd
```
Comment (ShadowDeath6) [inline code/game/objects/items/weapons/storage/backpack.dm]: need to add (src) at the end of the new stuff

**Comment by Kmc2000** [comment]
Inb4 mass drone abortion scandal to get infinite RCDS as tators

Literally china 🇨🇳🇨🇳🇨🇳🇨🇳

**Comment by Time-Green** [comment]
There's nothing wrong with drones. They don't necesarily need this.

**Comment by Zostroll** [comment]
Now there is gonna be soap everywhere.

**Comment by Super3222** [comment]
Drones don't need soap.
Drones don't need RCD's and RPD's. That just encourages them to fuck off and build an extension to the station.
Also, drones are in a good spot right now. There's no reason for RPD's and RCD's at all. They have the tools, they have the metal, and they have the glass. If you were expecting expecting everything to be easier when playing a ghost role ... well no, it's not.

Also you gave them the fucking syndie soap

Drones have a mechanical purpose and that's tied to repair. Just repair.

File: code/game/objects/items/weapons/storage/backpack.dm [hunk 2]
```diff
@@ -334,11 +334,15 @@
 
 	new /obj/item/weapon/screwdriver(src)
 	new /obj/item/weapon/wrench(src)
-	new /obj/item/weapon/weldingtool(src)
+	new /obj/item/weapon/weldingtool/hugetank(src)
 	new /obj/item/weapon/crowbar(src)
 	new /obj/item/stack/cable_coil/random(src)
 	new /obj/item/weapon/wirecutters(src)
 	new /obj/item/device/multitool(src)
+	new /obj/item/device/lightreplacer(src)
```
Comment (Super3222) [inline code/game/objects/items/weapons/storage/backpack.dm:342]: stealth

**Comment by ShadowDeath6** [comment]
> implying drones aren't syndicate scum

le mayo

**Comment by ghost** [comment]
syndie soap because it's fast

soap so that drones can clean

RPDs so that drones can fix broken piping

RCDs so that drones can fix breaches

it's not like it actually matters, drones just exist to repair, if they start getting in your way chances are they are breaking their laws, ahelp to get them banned on the spot

light replacer because broken lights

**Comment by Zostroll** [comment]
RCD's are meant to be rare if you can get one off a dead drone or say kill a drone it makes alot of steal objectives quite alot easier.

**Comment by Zostroll** [comment]
The light replacer is a good idea. You should give them a bottle of space cleaner or add a floor buffer underneath there body. Really anything that doesn't slip!

**Comment by Super3222** [comment]
> syndie soap because it's fast

You can do a better job by creating a special brand for drones, otherwise it ain't flying.

> soap so that drones can clean

They don't need to clean

> RPDs so that drones can fix broken piping
> 
> RCDs so that drones can fix breaches

They got metal and glass though. It's more of a hands-on experience.

> light replacer because broken lights

And that's the jaintors job as well

**Comment by ghost** [comment]
they don't have metal or glass
light replacing is still station repair there's no reason to deny it to drones since they'll probably steal one from engi anyway

and cleaning makes the station look nice :)

**Comment by Zostroll** [comment]
Adam maybe make drone soap that doesn't slip?

**Comment by BOBsHERE** [comment]
I like these ideas. Make the drones have their own non slip soap, and make the RCD's and stuff unusable on drone death.

**Comment by Kmc2000** [comment]
> they don't have metal or glass
> EVA does though, which they can freely access

Adam i thought you were against handholding....

Additionally, why would nt ship drones with a rival consortium's brand name, it makes no sense

**Comment by ghost** [comment]
drones don't fucking matter, it's not handholding

they just exist to repair

this isn't babymoding or affecting shit it's just making a boring ass ghost role a tiny bit more effective

and who the FUCK cares about SOAP

**Comment by BOBsHERE** [comment]
Its more that a stun at a bad time can completely affect the round.

**Comment by ghost** [comment]
drones should not be stunning people

**Comment by Zostroll** [comment]
They shouldn't. That doesn't mean they won't. Also the would be disposable drones dropping unwanted soap every were. We don't need a metric fuck ton of soap littering the halls.

**Comment by Alek2ander** [comment]
RCD, RPD and light replacers are for lazy humans.
Same with big welders. Drones should have no problems with boring repetitive tasks, so no automation for them please.

Soap slips, drones have a small inventory, so it's getting left on the floor eventually. Also if drones start with soap they're gonna clean runes and gang tags, because clearly it's my job if I have soap and if there is nobody standing next to something it's not important to any being, right?

**Comment by ShadowDeath6** [comment]
drones have a massive inventory actually

**Comment by Super3222** [comment]
This PR makes drones more of a problem than they already are. And considering we have a drone dispenser in Science, they are most likely being used/will be used.

Now anyway, moving back. Drones exist to repair, yes. But they should never overshadow **occupations within the game**. Which was one of /tg/'s problems with drones because frankly, they were too good. Adam, drones shouldn't be taking other peoples jobs, namely the Janitors, and at this point, you're buffing them to be even more considerably good.

So not only are you letting them overshadow Jaintors and Engineers but, you managed to conceive that it'd good be a good idea to let them take over the Atmospherics Jobs. I repeat one of my earlier statements from another PR, but at what cost are we doing these things?

If the excuse is, "Jaintors never fix the lights", well than that's a lie. It's like me saying that geneticists never clone people so why is their room only open to them. It's because that's apart of their job, and it's another responsibility on top of them. Whether they do their job or not is up to the people that play the game and, shouldn't be undermined by a random ghost role.

Also I'd like to note that drones fucking over shadowlings by replacing the lights IS A THING. And it's the worst experience to go through because you can't stun a drone.

This PR is bad and, the only thing feasible from it would be the welder. But even Alek2ander brings up a fair point. This is quite literally a drone hugbox.

And yes, I'm not the number one fan of drones. I did make a PR removing their dispenser but, in my defense it was because admins and players were disposing it. I do see use and efficiency of drones within the game so please, don't get me wrong. It's only your PR is just upping their game x2 and placing them in a terrible spot fucking over the balance. Which should not happen.

**Comment by Super3222** [comment]
This PR makes drones bigger problem than they already are And considering we have a drone dispenser in Science, we'll be seeing more and more of them because of what you're putting in Adam.

Now anyway, moving back. Drones exist to repair, yes. But you're forgetting something... they should never overshadow **occupations within the game**. Which was one of /tg/'s problems with drones because frankly, they were too good. Adam, drones shouldn't be taking other peoples jobs, namely the Janitors, and at this point, you're buffing them to be even more considerably good.

So not only are you letting them overshadow Jaintors and Engineers but, you managed to conceive that it'd good be a good idea to let them take over the Atmospherics Jobs. I repeat one of my earlier statements from another PR, but at what cost are we doing these things?

If the excuse is, "Jaintors never fix the lights", well than that's a lie. It's like me saying that geneticists never clone people so why is their room only open to them. It's because that's apart of their job, and it's another responsibility on top of them. Whether they do their job or not is up to the people that play the game and, shouldn't be undermined by a random ghost role.

Oh and neither is "drones fix things" an excuse either. Drones run and get boxes of light bulbs/tubes all the time.

Also I'd like to note that drones fucking over shadowlings by replacing the lights IS A THING. And it's the worst experience to go through because you can't stun a drone.

This PR is bad and, the only thing feasible from it would be the welder. But even Alek2ander brings up a fair point. This is quite literally a drone hugbox.

And yes, I'm not the number one fan of drones. I did make a PR removing their dispenser but, in my defense it was because admins and players were disposing it. I do see use and efficiency of drones within the game so please, don't get me wrong. It's only your PR is just upping their game x2 and placing them in a terrible spot fucking over the balance. Which should never happen.

And finally, I care about the soap.

**Comment by Super3222** [comment]
This PR makes drones an even more bigger problem than they already are. Considering we have a drone dispenser in Science, we'll be seeing more and more of them because of what you're putting in Adam.

Now anyway, moving back. Drones exist to repair, yes. But you're forgetting something... they should never overshadow occupations within the game. Which was one of /tg/'s problems with drones because frankly, they were too good. Adam, drones shouldn't be taking other peoples jobs, namely the Janitors, and at this point, you're buffing them to be even more considerably good.

So not only are you letting them overshadow Jaintors and Engineers but, you managed to conceive that it'd good be a good idea to let them take over the Atmospherics Jobs. I repeat one of my earlier statements from another PR, but at what cost are we doing these things?

If the excuse is, "Jaintors never fix the lights", well than that's a lie. It's like me saying that geneticists never clone people so why is their room only open to them. It's because that's apart of their job, and it's another responsibility on top of them. Whether they do their job or not is up to the people that play the game and, shouldn't be undermined by a random ghost role.

Oh and neither is "drones fix things" an excuse either. Drones run and get boxes of light bulbs/tubes all the time.

Also I'd like to note that drones fucking over shadowlings by replacing the lights IS A THING. And it's the worst experience to go through because you can't stun a drone.

This PR is bad and, the only thing feasible from it would be the welder. But even Alek2ander brings up a fair point. This is quite literally a drone hugbox.

And yes, I'm not the number one fan of drones. I did make a PR removing their dispenser but, in my defense it was because admins and players were disposing it. I do see use and efficiency of drones within the game so please, don't get me wrong. It's only your PR is just upping their game x2 and placing them in a terrible spot fucking over the balance. Which should never happen.

And finally, I care about the soap.

**Comment by ghost** [comment]
okay i made the nonslippery soap

here are a few things

what do janitors do? guess what-it's not clean

they usually just suicide and on the rare occasions taht they do do their job i doubt they'll mind the extra help

what do janitors NEVER do?

replace lights

what is the issue with drones? If they're getting in your way adminhelp to get them banned, a drone is literally incapable of doing anything whatsoever to impede you in its lawset

if it replaces a light near a shadowling it gets banned, if it crowbars up a blood rune it gets banned, if it dissasembles a plasma flood or any of that it gets banned

it repairs and improves. it can't space a syndiebomb before it dets, it has to wait for it to det to repair

seriously, if a drone gets in your way adminhelp to get it banned on the spot. if there's so much as one attack log on the part of the drone it will get banned, no questions asked

**Comment by Dagdammit** [comment]
Drones don't need soap because they have crowbars. Non-slip soap would be fine.

RPD seems potentially acceptable, light replacer seems unnecessary but potentially acceptable, RCD no way son.

**Comment by ghost** [comment]
cruix i may or may not literally die please pity merge my PR if i do

**Pull request closed without merging**